### DISCUSSION TITLE
YugabyteDB as a backend database option

### DISCUSSION THREAD

**Discussion by ymahajan** [twentyhq/twenty#6586] [Ideas]
We have tested your app on [YugabyteDB](https://docs.yugabyte.com/) and it works fine. We can provide the instruction set to install YugabyteDB, which should be simple. How do you suggest we should put in a request to update relevant documentation on your end?

**Comment by FelixMalfait** [comment]
Hey @ymahajan thanks a lot for the suggestion! I think for now we'll focus on just supporting Postgres to avoid confusing users too much.
